src/main/java/de/tum/cit/aet/artemis/programming/service/ci/ContinuousIntegrationService.java

```diff
@@ -263,6 +263,77 @@ interface CustomizableCheckoutPath {
         String forProgrammingLanguage(ProgrammingLanguage language);
     }
 
+    enum DependencyDownloadScript {
```

Comment:
Should we move this into its own file? This makes up roughly one third of the service